src/components/settings/SpendingLimits/NewSpendingLimit/index.tsx

```diff
@@ -30,11 +31,16 @@ export const NewSpendingLimit = () => {
 
   return (
     <>
-      <Track {...SETTINGS_EVENTS.SPENDING_LIMIT.NEW_LIMIT}>
-        <Button onClick={() => setOpen(true)} sx={{ marginTop: 2 }} variant="contained">
-          New spending limit
-        </Button>
-      </Track>
+      <CheckWallet>
+        {(isOk) => (
+          <Track {...SETTINGS_EVENTS.SPENDING_LIMIT.NEW_LIMIT}>
+            <Button onClick={() => setOpen(true)} sx={{ marginTop: 2 }} variant="contained" disabled={!isOk}>
+              New spending limit
+            </Button>
+          </Track>
+        )}
+      </CheckWallet>
+
```

Comment:
The tooltip position looks off because the `Track` wrapper takes up the full width
<img width="846" alt="Screenshot 2023-03-02 at 10 42 55" src="https://user-images.githubusercontent.com/5880855/222391497-da8e6fa9-507a-4599-b03f-276530e45a5d.png">